app/models/pageflow/theming.rb

```diff
@@ -26,5 +26,17 @@ def theme
     def name
       I18n.t('pageflow.admin.themings.name', :account_name => account.name, :theme_name => theme_name)
     end
+
+    def copy_defaults_to(revision)
+      widgets.all.each do |widget|
+        widget.copy_to(revision)
+      end
+
+      revision.update(
+        author: default_author.presence || Pageflow.config.default_author_meta_tag,
+        publisher: default_publisher.presence || Pageflow.config.default_publisher_meta_tag,
+        keywords: default_keywords.presence || Pageflow.config.default_keywords_meta_tag
+      )
+    end
```

Comment:
I think Hound is right here. Let's keep the `Widget.copy_all_to` and extract the revision update into `copy_default_meta_tags` method. That way `copy_defaults_to` has a nicer abstraction level.